packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -452,9 +416,7 @@ class InspectorController extends DisposableController
     treeGroups.cancelNext();
     try {
       final group = treeGroups.next;
-      final node = await (detailsSubtree
-          ? group.getDetailsSubtree(subtreeRoot, subtreeDepth: subtreeDepth)
-          : group.getRoot(treeType, isSummaryTree: false));
+      final node = await group.getRoot(treeType, isSummaryTree: false);
```

Comment:
can we remove the isSummaryTree param from this method?

Reply:
I made the default value for the parameter `false`, but we can't remove it because these inspector service methods are shared with the legacy inspector.